edgelet/kube-client/src/lib.rs

```diff
@@ -18,3 +18,109 @@ pub mod kube;
 pub use self::client::{Client, HttpClient};
 pub use self::config::{get_config, Config, TokenSource, ValueToken};
 pub use self::error::{Error, ErrorKind, RequestType};
+
+#[cfg(test)]
+#[allow(dead_code)]
```

Comment:
should we remove that?

Reply:
There is some dead code I wound up not using for my tests. I figured it would be more valuable to leave in the CertGenerator options in.